developer/variables.boot.volume.pkrvars.hcl

```diff
@@ -0,0 +1,6 @@
+SUBNET_ID="0727-9447a2c3-efb8-44d2-a180-115531e0b0d3"
+REGION="us-south"
+SECURITY_GROUP_ID=""
+RESOURCE_GROUP_ID=""
+IBM_API_KEY="UUXUGZf3BSWd-kVJHLqBJVfirD2xPphT8ecK8nTbj-Jg"
```

Comment:
IBM_API_KEY is exposed here. If this is a real API KEY, then you should revoke the API KEY. 
For the future, run `detect-secrets` as `pre-commit` hook.

Reply:
revoked this change